### PR TITLE
Support const array of gdk::Atom

### PR DESCRIPTION
**Pull request by EPashkin**
Will be needed for https://github.com/gtk-rs/gir-files/pull/32, but can be done before.

cc @GuillaumeGomez , @sdroege

### CODE REVIEW

File: src/atom.rs [hunk 1]
```diff
@@ -113,6 +113,44 @@ impl<'a> ToGlibContainerFromSlice<'a, *mut ffi::GdkAtom> for &'a Atom {
...
+        (v_ptr.as_ptr() as *const ffi::GdkAtom, (v, Some(v_ptr)))
+    }
+
+    fn to_glib_container_from_slice(t: &'a [&'a Atom]) -> (*const ffi::GdkAtom, Self::Storage) {
+        skip_assert_initialized!();
+
+        let v: Vec<_> = t.iter().map(|s| s.to_glib_none()).collect();
+
+        let v_ptr = unsafe {
+            let v_ptr = glib_ffi::g_malloc0(mem::size_of::<ffi::GdkAtom>() * t.len() + 1) as *mut ffi::GdkAtom;
```
Comment (GuillaumeGomez) [inline src/atom.rs]: I think the correct syntax here would be (wrapping the `t.len() + 1`):

```
let v_ptr = glib_ffi::g_malloc0(mem::size_of::<ffi::GdkAtom>() * (t.len() + 1)) as *mut ffi::GdkAtom;
```

Comment (EPashkin) [reply src/atom.rs]: Good catch, thanks

Comment (EPashkin) [reply src/atom.rs]: Fixed

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**